packages/zipkin/src/time.js

```diff
@@ -1,4 +1,12 @@
 // Returns the current time in epoch microseconds
-module.exports.now = function now() {
-  return new Date().getTime() * 1000;
+// if startTimestamp and startTick are present, process.hrtime is used
+// See https://nodejs.org/api/process.html#process_process_hrtime_time
+module.exports.now = function now(startTimestamp, startTick) {
+  if (startTimestamp && startTick && process && process.hrtime) {
```

Comment:
takes me a very long time to figure out how to do things like this as I'm not very experienced in javascript. Can you give me an example of how to conditionally initialize now like you mentioned?